app/src/main/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentService.java

```diff
@@ -67,8 +67,6 @@ public JsonObject getTsnePlotData(String experimentAccession) {
         tsnePlotSettingsService.getExpectedClusters(experimentAccession)
                 .ifPresent(value -> result.addProperty("selectedK", value));
 
-        result.add("perplexities", getPerplexities(experimentAccession));
```

Comment:
Please modify the test for this.

Reply:
I missed it! Will do, many thanks.

Reply:
I leave this as it is, since this is fixed in the upcoming PR with the new cell plots endpoint. Well spotted!